test/visualTest.ts

```diff
@@ -156,7 +156,7 @@ describe("ChicletSlicer", () => {
                 const containerHeight: number = Number(visualBuilder.slicerItemContainer
                     .clientHeight);
 
-                const slicerFontSize: number = Number(<HTMLElement>(visualBuilder.slicerItemContainer
+                const slicerFontSize: number = Number((<HTMLElement>visualBuilder.slicerItemContainer
```

Comment:
explain this please

Reply:
we must case result of querySelector to HTMLElement before we can access its 'style' property